07_wait_for_cluster.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -x
+set -e
+
+source logging.sh
+source common.sh
+
+export KUBECONFIG=ocp/$CLUSTER_NAME/auth/kubeconfig
+
+workers=$NUM_WORKERS
+if $APPLY_EXTRA_WORKERS ; then
+    workers=($workers + $NUM_EXTRA_WORKERS)
+fi
+if (( $workers <= 0 )); then
+    # we need some workers to have all the operators up.
+    exit 0
```

Comment:
I don't think this is correct, with `NUM_WORKERS=0` the controlplane is schedulable, so the operators should all still come up, just some services e.g ingress end up on the masters instead.

Reply:
ok, let me check again. I might have been looking at the wrong condition..